src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -262,7 +262,8 @@ public int execute(String... args)
                .withCommand(CMSAdmin.InitializeCMS.class)
                .withCommand(CMSAdmin.ReconfigureCMS.class)
                .withCommand(CMSAdmin.Snapshot.class)
-               .withCommand(CMSAdmin.Unregister.class);
+               .withCommand(CMSAdmin.Unregister.class)
+               .withCommand(AccordAdmin.MarkStaleCmd.class);
```

Comment:
This shouldn't be here

Reply:
Yeah, forgot to remove when I moved `AccordAdmin` to its own group.